### PR TITLE
New questions batch

### PR DESCRIPTION
**Pull request by seregazhuk**
I'v added new questions for your pack.

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
Hi @seregazhuk,

thank you for your contribution.  I will review asap. 

Mickaël

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.DS_Store
+.idea
```
Comment (xabbuh) [inline .gitignore]: You should add these files to your global ignores rather than committing them to this repo.

Comment (seregazhuk) [inline .gitignore]: Ok, I'll remove it from request.

**Comment by mickaelandrieu** [comment]
thank you @seregazhuk  (and thank you @xabbuh for review)

**Pull request merged**